### PR TITLE
Pin Wasmer version to prevent test failures

### PR DESCRIPTION
**Pull request by MaxDesiatov**
Apparently, Wasmer 2.2.0-rc1 is causing our tests to fail, let's pin it to 2.1.1 on both Linux and macOS.

### CODE REVIEW

File: utils/update_checkout/update-checkout-config.json [hunk 1]
```diff
@@ -93,6 +93,8 @@
                 "swift-tools-support-core": "main",
                 "swiftpm": "main",
                 "swift-argument-parser": "1.0.1",
+                "swift-atomics": "1.0.2",
+                "swift-collections": "1.0.1",
```
Comment (MaxDesiatov) [inline utils/update_checkout/update-checkout-config.json:97]: Something's changed upstream that made these projects required now, their lack causes a build breakage.

**Review by kateinoigakukun — APPROVED** [review]

**Comment by syrusakbary** [comment]
@MaxDesiatov @kateinoigakukun which tests are failing so we can take a look? / cc @Amanieu @ptitSeb

**Comment by kateinoigakukun** [comment]
@syrusakbary reported 👍

**Comment by syrusakbary** [comment]
Thanks!

**Pull request merged**